content/en/docs/contribute/style/content-guide.md

```diff
@@ -45,68 +45,68 @@ effort from project maintainers and tends to become outdated more quickly.
-        - Linking to active CNCF projects
-            - Example: linking to the Prometheus [documentation](https://prometheus.io/docs/introduction/overview/); Prometheus is an active CNCF project
-    - Not Allowed:    
-        - Linking to vendor-specific products
-        - Linking to archived CNCF projects
-        - Linking to inactive projects in the kubernetes and kubernetes-sigs GitHub organizations
-        - Linking to open source projects that are not CNCF projects or do not reside in the kubernetes or kubernetes-sigs GitHub organizations
+    1. Allowed:    
+        1. Linking to projects in the kubernetes and kubernetes-sigs GitHub organizations
+            1. Example: linking to Kubernetes in Docker (KinD) [documentation](https://kind.sigs.k8s.io/docs/user/quick-start), which resides in the kubernetes-sigs GitHub organization
+        1. Linking to active CNCF projects
+            1. Example: linking to the Prometheus [documentation](https://prometheus.io/docs/introduction/overview/); Prometheus is an active CNCF project
+    1. Not Allowed:    
+        1. Linking to vendor-specific products
+        1. Linking to archived CNCF projects
+        1. Linking to inactive projects in the kubernetes and kubernetes-sigs GitHub organizations
+        1. Linking to open source projects that are not CNCF projects or do not reside in the kubernetes or kubernetes-sigs GitHub organizations
 1. Content about training courses
-    - Allowed:    
-        - Linking to vendor-neutral Kubernetes training courses offered by the [CNCF](https://www.cncf.io/), the [Linux Foundation](https://www.linuxfoundation.org/), and the [Linux Academy](https://linuxacademy.com/), which is a partner of the Linux Foundation
-            - Example: linking to Linux Academy courses such as [Kubernetes Quick Start](https://linuxacademy.com/course/kubernetes-quick-start/) and [Kubernetes Security](https://linuxacademy.com/course/kubernetes-security/)      
-    - Not Allowed:    
-        - Linking to online training outside of the CNCF, the Linux Foundation, or the Linux Academy; the Kubernetes documentation does not link to third-party content 
-            - Example: linking to Kubernetes tutorials or courses on Medium, KodeKloud, Udacity, Coursera, learnk8s, and similar websites
-        - Linking to vendor-specific tutorials regardless of the training provider
-            - Example: linking to Linux Academy courses such as [Google Kubernetes Engine Deep Dive](https://linuxacademy.com/google-cloud-platform/training/course/name/google-kubernetes-engine-deep-dive) and [Amazon EKS Deep Dive](https://linuxacademy.com/course/amazon-eks-deep-dive/)
+    1. Allowed:    
+        1. Linking to vendor-neutral Kubernetes training courses offered by the [CNCF](https://www.cncf.io/), the [Linux Foundation](https://www.linuxfoundation.org/), and the [Linux Academy](https://linuxacademy.com/), which is a partner of the Linux Foundation
```

Comment:
nit: Which Linux Academy courses are allowed? It is confusing that you provide Linux Academy examples below that are not allowed

Reply:
This will be done in a separate PR.  I've reverted my content-guide.md changes.